config/all-in-one/templates/cp4waiops.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.cp4waiops.aiManager.enabled }}
```

Comment:
Suggest to rename for consistency as below:
- File name `cp4waiops.yaml` to be `aimanager.yaml`
- ApplicationSet `metadata.name` to be `cp4waiops-aimanagers`
- Application `metadata.name` to be `{{name}}-cp4waiops-aimanager`

Reply:
Done